.github/workflows/pr-check.yaml

```diff
@@ -164,16 +164,16 @@ jobs:
         run: pnpm install
 
       - name: Install SSO extension dependencies
-        if: matrix.os == 'windows-2022'
+        if: matrix.os == 'windows-2022' || matrix.os == 'macos-14'
```

Comment:
Let's install and build every dependency/extension/podman-desktop at once (where possible) to reduce number of steps.